zitadel/v2/helper/client.go

```diff
@@ -26,8 +28,15 @@ type ClientInfo struct {
 	Options []zitadel.Option
 }
 
-func GetClientInfo(insecure bool, domain string, token string, port string) (*ClientInfo, error) {
-	options := []zitadel.Option{zitadel.WithJWTProfileTokenSource(middleware.JWTProfileFromPath(token))}
+func GetClientInfo(insecure bool, domain string, jwtProfileFile string, jwtProfileJSON string, token string, port string) (*ClientInfo, error) {
+	options := []zitadel.Option{}
+	if jwtProfileFile != "" {
+		options = append(options, zitadel.WithJWTProfileTokenSource(middleware.JWTProfileFromPath(jwtProfileFile)))
```

Comment:
Why add an attribute which has the same functionality as "token"? Is there any reason for it?

Reply:
@stebenz 
This is just for compatibility with the old one written in the description above.
I'll remove it if you don't require

> token and jwt_profile_file are exactly the same thing but I the old one to keep compatibility.
> If the alpha doesn't have to support backward compatibility, let's remove it.

Reply:
No that's fine by me as additional. We can remove the old part when v1.0.0 is created.